docker/dev/docker-compose.yml

```diff
@@ -35,13 +42,19 @@ services:
       - RUST_LOG="warn,lemmy_server=debug,lemmy_api=debug,lemmy_api_common=debug,lemmy_api_crud=debug,lemmy_apub=debug,lemmy_db_schema=debug,lemmy_db_views=debug,lemmy_db_views_actor=debug,lemmy_db_views_moderator=debug,lemmy_routes=debug,lemmy_utils=debug,lemmy_websocket=debug"
       - RUST_BACKTRACE=full
     volumes:
-      - ./lemmy-dev.hjson:/config/config.hjson
+      - ./lemmy.hjson:/config/config.hjson
     depends_on:
       - postgres
       - pictrs
 
   lemmy-ui:
     image: dessalines/lemmy-ui:dev
+    # use this to build your local lemmy server image for development
+    # checkout git repo to build and
+    # run docker compose up --build
+    #build: 
+    #  context: ./build/lemmy-ui
+    #  dockerfile: ./build/lemmy-ui/docker/dev/Dockerfile  
```

Comment:
Same here, if lemmy-ui repo is cloned next to lemmy repo, should work more or less like this:
```
build:
  context: ../../../lemmy-ui
  dockerfile: ../../../lemmy-ui/Dockerfile  
```

Reply:
True.